### DISCUSSION TITLE
Stats to show most read and bookmarked feeds

### DISCUSSION THREAD

**Discussion by Danie10** [FreshRSS/FreshRSS#6806] [Ideas]
When pruning down on my feeds, a very useful way to do it is to look at the feeds that I have most read articles from vs least read (or bookmarked). In this way, I can consider ditching the feeds I don't find articles of interest.

Current stats show frequency / number of articles per feed, but that is just an indication of how busy a feed is, and when.

If I could see a percentage actually read / bookmarked per feed, it could show me the ones I found the most interesting.  The count per feed is not the most important as some feeds may be very quiet, but I read 100% of its posts.

The only snag may be is I browse down a feed and then mark it as Read before moving to the next feed. I'm not sure if that will throw things off, as marking as read does not mean opening or actually reading the article? Is there a better way of marking a feed as "done"?

I see there was an issue opened at https://github.com/FreshRSS/FreshRSS/issues/4051 which looked similar, but that was closed without any comments.

**Comment by math-GH** [comment]
> I see there was an issue opened at #4051 which looked similar, but that was closed without any comments.

This linked issue is not closed and is still open. 
There is  #4052 linked that was closed by the PR opener.

**Comment by nfnfgo** [comment]
I think it’s a good suggestion to enable users to know what kinds of feeds they are more likely to focus on. 

Adding a completely new operations and require users to manually mark an entry as “done” seems to be a little bit too costly if it’s just for showing stat.

It might be a better choice that FreshRSS just simply **record the action of user when user open an entry**. And I think something like “total open count” could a good index when evaluating the value of a feed for a certain user. _(Indeed, there could be cases like you opened an entries to see description and found yourself not interested in this one, but in terms of stat calculating, entry-opened action should be enough to reflect the importance of a feed/category/label etc.)_

Then based on this data, FreshRSS could provide users with stats like “Most opened feeds/categories/labels” etc.
